pulp_2to3_migration/app/migrations/0024_errata_to_subid.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.19 on 2021-03-09 01:47
+
+from django.db import migrations, models
+from django.db.models import Count, F
+
+
+def fix_dups(apps, schema_editor):
+    pulp2content = apps.get_model("pulp_2to3_migration", "Pulp2Content")
+    # find sets of p2-id/p2-type/-2-repo/p2-sub dups
+    p2content = pulp2content.objects.values('pulp2_id', 'pulp2_content_type_id', 'pulp2_repo_id', 'pulp2_subid').order_by('pulp2_id')
+    p2content = p2content.annotate(count_id=models.Count("pulp2_id")).filter(count_id__gt=1)
+
+    # for each set of dups...
+    for dup in p2content:
+        # ...find the pulp-ids of entries in that set, order by pulp2-last-update
+        to_delete = pulp2content.objects.filter(**{x: dup[x] for x in ['pulp2_id', 'pulp2_content_type_id', 'pulp2_repo_id', 'pulp2_subid']}).order_by('pulp2_last_updated')
+        # exclude the first/newest one...
+        to_delete = to_delete.exclude(pk=to_delete[0].pk)
+        # ...and delete the rest
+        to_delete.delete()
+
+    # for existing errata, copy repo_id to subid to maintain uniqueness
+    errata = pulp2content.objects.filter(pulp2_content_type_id='erratum').update(pulp2_subid=F('pulp2_repo_id'))
```

Comment:
With the 'correct' UQ (if repo-id is null we demand that p2-id/p2-content-type-id/p2-subid is unique), we may not need to duplicate repo-id into subid. Leaving this here awaiting further discussion.

Reply:
After discussion agreement that we don't need to touch errata-entries - removed.